src/xtgeo/grid3d/_gridprop_import.py

```diff
@@ -30,78 +32,62 @@ def from_file(
     if not isinstance(pfile, xtgeo._XTGeoFile):
         raise RuntimeError("Internal error, pfile is not a _XTGeoFile instance")
 
-    fformat = _chk_file(self, pfile.name, fformat)
+    pfile.check_file(raiseerror=OSError)
+
+    if fformat is None or fformat == "guess":
+        fformat = pfile.detect_fformat(suffixonly=True)
+    fformat = fformat.lower()
 
-    if fformat == "roff":
+    if fformat in ["roff", "roff_binary", "roff_ascii"]:
```

Comment:
Should this not be ["roff_ascii", "roff_asc", "roff-asc", "roffasc", "asc"], and in that case should it not fetch the supported list from SUPPORTED_FORMATS? This goes for all fformat checks

Reply:
These would be changed in PR #662 in any case, I stole the logic from there a bit :)

Reply:
Alright, then I trust that it will be taken care off later :)